tsconfig.json

```diff
@@ -44,7 +44,7 @@
     "src/index.cjs",
     "standalone.js",
     "bin/",
-    "dist/",
+    "dist*/",
```

Comment:
Is this intended?

Reply:
Yes, when debugging build script, I save different `dist` directories, so I ignored `dist*` https://github.com/prettier/prettier/blob/92a6af22e72cf1d937492fecc591bb41edd16306/.gitignore#L14